golang/cosmos/x/swingset/keeper/extension_snapshotter.go

```diff
@@ -298,10 +293,9 @@ func (snapshotter *ExtensionSnapshotter) RestoreExtension(blockHeight uint64, fo
 	// At this point the content of the cosmos DB has been verified against the
 	// AppHash, which means the SwingStore data it contains can be used as the
 	// trusted root against which to validate the artifacts.
-	getExportData := func() ([]*vstoragetypes.DataEntry, error) {
+	getExportDataIterator := func() (sdk.Iterator, error) {
 		ctx := snapshotter.newRestoreContext(height)
```

Comment:
I'll add a comment that `newRestoreContext` is just to mint and `sdk.Context` that carries the `BlockHeight` information expected by the KVStore

Reply:
This probably just warrants a comment that we want construction of the iterator to be lazy in case the specific SwingStoreRestoreOptions render it unnecessary.